How to change the 'time' parameter of the deduplication handler

ThomasLandauer opened the pull request:
Don't know how this needs to be written in XML & PHP, sorry.

ThomasLandauer committed:
How to change the 'time' parameter of the deduplication handler

Don't know how this needs to be written in XML & PHP, sorry.

xabbuh committed:
add XML and PHP config examples

xabbuh reviewed (approved):
I just pushed an example for the XML and PHP format.

javiereguiluz commented on logging/monolog_email.rst:
```diff
@@ -28,6 +28,8 @@ it is broken down.
                     handler:      deduplicated
                 deduplicated:
                     type:    deduplication
+                    # uncomment this line for values other than 60 seconds:
```
This comment and the provided `time: 10` example looks a bit confusing to me. What if we explain this option more in detail and don't include the `time: 10` example commented?

```yaml
deduplicated:
    type:    deduplication
    # the time in seconds during which duplicate entries are discarded (default: 60)
    time: 10
    handler: swift
```

HeahDude replied:
I like @javiereguiluz suggestion.

ThomasLandauer commented:
@javiereguiluz Don't know what you mean exactly.

My motivation for committing this: Right now, the `time` setting is explained nowhere. I basically guessed how to write it ;-) The existence of the setting is mentioned a few paragraphs further down:

> The deduplicated handler simply keeps all the messages for a request and then passes them onto the nested handler in one go, but only if the records are unique over a given period of time (60 seconds by default).

So I figured I could squeeze it right into the first code block. Not ideal, but better than nothing :-)

However, we could certainly create a separate code block for this, maybe something like:
````
monolog:
    handlers:
        deduplicated:
            type:    deduplication
            handler: swift
            time: 10

... and position it right below the cited paragraph.

ThomasLandauer committed:
`time` option

As suggested by https://github.com/symfony/symfony-docs/pull/7896#pullrequestreview-38006819

XML and PHP syntax is taken from https://github.com/symfony/symfony-docs/pull/7896/files/9d2f2fd9df136b63275e0ce991196e9dda213f7d

ThomasLandauer committed:
Update monolog_email.rst

Removing the now obsolete comments of https://github.com/symfony/symfony-docs/pull/7896/files/9d2f2fd9df136b63275e0ce991196e9dda213f7d
